test/inductor/test_b2b_gemm.py

```diff
@@ -0,0 +1,28 @@
+import torch
+
+from torch._inductor.test_case import run_tests, TestCase
+
+class B2BGEMMTest(TestCase):
+
+    @torch._inductor.config.patch(b2b_gemm_pass=True)
+    def test_b2b_gemm_1(self):
```

Comment:
nit: maybe give these a bit more descriptive names like `test_b2b_gemm_supported` and `test_b2b_gemm_unsupported`?

Reply:
Changed.